borsh-rs/borsh/tests/test_simple_structs.rs

```diff
@@ -9,7 +9,9 @@ struct A<'a> {
     y: f32,
     z: String,
     t: (String, u64),
+    #[cfg(feature = "std")]
```

Comment:
Can we also have the tests for the hashmap and other types we support in no-std context?

Reply:
done